### ISSUE TITLE
Getting a bunch of Module not found errors after upgrading project from cli 1.6.1 to 1.7.0-rc.0

### ISSUE THREAD

**Issue by abeymg** [angular/angular-cli#9575]
### Versions
```
Angular CLI: 1.7.0-rc.0
Node: 6.9.0
OS: darwin x64
Angular: 5.2.4
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, platform-server, router

@angular/cdk: 5.2.0
@angular/cli: 1.7.0-rc.0
@angular/flex-layout: 2.0.0-beta.12
@angular/material: 5.2.0
@angular-devkit/build-optimizer: 0.3.1
@angular-devkit/core: 0.3.1
@angular-devkit/schematics: 0.0.34
@ngtools/json-schema: 1.1.0
@ngtools/webpack: 1.10.0-rc.0
typescript: 2.5.3
webpack: 3.10.0
```

### Repro steps
* Step 1
Upgrade an angular project using cli version 1.6.1 to cli 1.7.0-rc.0. App uses the pubnub-angular2 lib v 1.3.0 (https://www.npmjs.com/package/pubnub-angular2)

* Step 2
ng serve the app OR alternatively ng test any specs that make use the pubnub angular service.

### Observed behavior
Getting a bunch of Module not found error e.g.
```
ERROR in ./node_modules/pac-proxy-agent/node_modules/socks-proxy-agent/index.js
Module not found: Error: Can't resolve 'dns' in '/Users/ag/proj/node_modules/pac-proxy-agent/node_modules/socks-proxy-agent'
ERROR in ./node_modules/pac-resolver/dnsResolve.js
Module not found: Error: Can't resolve 'dns' in '/Users/ag/proj/node_modules/pac-resolver'
ERROR in ./node_modules/pac-resolver/isInNet.js
Module not found: Error: Can't resolve 'dns' in '/Users/ag/proj/node_modules/pac-resolver'
ERROR in ./node_modules/pac-resolver/isResolvable.js
Module not found: Error: Can't resolve 'dns' in '/Users/ag/proj/node_modules/pac-resolver'
ERROR in ./node_modules/socks-proxy-agent/socks-proxy-agent.js
Module not found: Error: Can't resolve 'dns' in '/Users/ag/proj/node_modules/socks-proxy-agent'
```

### Desired behavior
Should not throw module not found errors.

### Mention any other details that might be useful (optional)
I realize this may be an issue with the library, but its unclear to me why a new cli version would affect this.

**Comment by abeymg** [comment]
Is this fix included in the 1.7.0 release published to npm?

**Comment by abeymg** [comment]
I just updated to cli 6.0.0-beta.3 and can confirm that this is not an issue anymore.
